insalan/cms/tests.py

```diff
@@ -23,3 +24,16 @@ def test_content_with_unknow_constants(self):
     def test_display_undefined_constant_list(self):
         Content.objects.create(name="content", content="je test ${inconnue} mais aussi une variable ${random}")
         self.assertRaisesMessage(ValidationError, _("Des constantes non définies sont utilisées: inconnue, random"))
+
+    def test_create_two_contents_with_same_name(self) -> None:
+        with self.assertRaises(IntegrityError):
+            Content.objects.create(name="content", content="content")
+            Content.objects.create(name="content", content="content")
+
+
+class ConstantTestCase(TestCase):
+
+    def test_create_two_constants_with_same_name(self) -> None:
+        with self.assertRaises(IntegrityError):
+            Constant.objects.create(name="const", value="1")
+            Constant.objects.create(name="const", value="1")
```

Comment:
maybe just for the sake of not missing something obvious these two (and the two above) should also have different *values*

Reply:
fixed in #0dd7c